spec/lib/common/system-uuid-spec.js

```diff
@@ -0,0 +1,43 @@
+// Copyright 2016, EMC, Inc.
+'use strict';
+
+describe("SystemUuid", function() {
+    var subject;
+    var regEx = /^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$/;
```

Comment:
It is the same as assert-plus's assert.uuid().   https://github.com/mcavage/node-assert-plus/blob/master/assert.js#L12

Are you suggesting we're not asserting UUID values correctly and shouldn't be using assert.uuid()?